src/engine/source/kvdb2/include/kvdb2/kvdbManagedHandler.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef _KVDBMANAGEDHANDLER_H
```

Comment:
I think it should be _KVDB_MANAGED_HANDLER_H

Reply:
Will be fixed in later documentation step.